go.mod

```diff
@@ -1,92 +1,65 @@
 module github.com/kubeflow/pipelines
 
 require (
-	cloud.google.com/go v0.44.3 // indirect
 	github.com/Masterminds/squirrel v0.0.0-20190107164353-fa735ea14f09
 	github.com/VividCortex/mysqlerr v0.0.0-20170204212430-6c6b55f8796f
-	github.com/argoproj/argo v2.3.0+incompatible
+	github.com/argoproj/argo v0.0.0-20200506223611-54154c61eb4f
```

Comment:
Is it possible to use a release version here?

Reply:
Not sure, will check if possible.

Reply:
The answer seems to be no, @Ark-kun https://github.com/argoproj/argo/issues/2602#issuecomment-636097428

Reply:
I meant to ask whether it' possible to refer to particular GIT tag like `github.com/argoproj/argo v2.7.5...`

Reply:
I will try to clean this up after I fixed some of the bugs :)

Reply:
I don't get it to work to specify the version as you mention @Ark-kun, agree that it would be better to have it with version.